Resources/Prototypes/Entities/Mobs/NPCs/xeno.yml

```diff
@@ -328,6 +334,24 @@
     spawned:
     - id: FoodMeatRouny
       amount: 3
+      
+- type: entity
+  name: Friendshaped
+  parent: MobXenoRunnerNPC # Delta V - These no longer inherit the ghost takeover comp
+  id: MobFriendshapedNPC # Delta V - These no longer inherit the ghost takeover comp
+  components:
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: Mobs/Aliens/Xenos/rouny.rsi
+    offset: 0,0.6
+  - type: NpcFactionMember
+    factions:
+    - PetsNT
+  - type: Butcherable
+    butcheringType: Spike
+    spawned:
+    - id: FoodMeatRouny
+      amount: 3
```

Comment:
All of those will be need to be moved to Floof/Mobs/NPCs/xeno.yml

Reply:
I've done the move, and I even altered it like you said- but it's still throwing an error